etc/submit_py3.sh

```diff
@@ -211,21 +211,27 @@ $pythonCommand Startup.py
 jobrc=$?
 echo -e "======== WMAgent Run the job FINISH at $(TZ=GMT date) ========\n"
 
+echo "WMAgent bootstrap: WMAgent finished the job, it's copying the pickled report"
+set -x
+cp WMTaskSpace/Report*.pkl ../
+ls -l WMTaskSpace
+ls -l WMTaskSpace/*
+
 echo "======== WMAgent add timestamps metrics to FJR $outputFile ========"
+cd ../
 timeEndSec=`date +%s`
-reportIn=$outputFile
-$pythonCommand Timestamps.py --reportFile=$outputFile --wmJobEnd=$timeEndSec --wmJobStart=$timeStartSec
+echo "WMAgent adding WMTiming metrics"
+tpy=/srv/job/Utils/Timestamps.py
+echo "======== WMAgent add timestamps metrics to FJR $outputFile ========"
+echo -e "execute $pythonCommand $tpy --reportFile=$outputFile --wmJobEnd=$timeEndSec --wmJobStart=$timeStartSec"
+$pythonCommand $tpy --reportFile=$outputFile --wmJobEnd=$timeEndSec --wmJobStart=$timeStartSec
```

Comment:
I would actually suggest to use it like `--reportFile=WMTaskSpace/$outputFile`

Reply:
I made necessary changes using `$HOME` environment